composeml/label_maker.py

```
@@ -92,11 +109,16 @@ def df_to_labels(df, progress_bar):
                 df = df[cutoff_time:]
 
                 if df.empty:
+                    # TODO Also apply when cutoff time is None.
                     skipped_iterations = num_examples_per_instance - example
                     progress_bar.update(n=skipped_iterations)
                     break
 
                 window_end = offset_time(df.index, self.window_size)
+
+                if window_in_parameters:
+                    kwargs['window'] = (cutoff_time, window_end)
```

should raise an error if the user has passed window since it is a reserved word